Formula/libmagic.rb

```diff
@@ -1,8 +1,8 @@
 class Libmagic < Formula
   desc "Implementation of the file(1) command"
   homepage "https://www.darwinsys.com/file/"
-  url "https://astron.com/pub/file/file-5.39.tar.gz"
-  sha256 "f05d286a76d9556243d0cb05814929c2ecf3a5ba07963f8f70bfaaa70517fad1"
+  url "https://astron.com/pub/file/file-5.40.tar.gz"
+  sha256 "167321f43c148a553f68a0ea7f579821ef3b11c27b8cbe158e4df897e4a5dd57"
 
```

Comment:
```suggestion
  # libmagic has a BSD-2-Clause-like license
  license :cannot_represent

```
https://metadata.ftp-master.debian.org/changelogs//main/f/file/file_5.39-3_copyright

Also, from `COPYING`:
```
Redistribution and use in source and binary forms, with or without
modification, are permitted provided that the following conditions
are met:
1. Redistributions of source code must retain the above copyright
   notice immediately at the beginning of the file, without modification,
   this list of conditions, and the following disclaimer.
2. Redistributions in binary form must reproduce the above copyright
   notice, this list of conditions and the following disclaimer in the
   documentation and/or other materials provided with the distribution.
```

Reply:
This is good. `libmagic` and `file-formula` are from the same sources. But `file-formula` has a license `BSD-2-Clause`. Is this right?

Reply:
`file-formula` probably needs to be fixed, then, as it seems `BSD-2-Clause` is actually slightly more permissive than the actual license. (A formula that states a less permissive license than the correct one is mostly fine, but one that is more permissive is bad.)